source/Cosmos.IL2CPU/CosmosAssembler.cs

```diff
@@ -388,7 +395,10 @@ public void Initialize(bool enableVBE, string VBEResolution)
 
             //Initiate Memory
             WriteDebugVideo("Initiating Memory");
-            XS.Call(LabelName.Get(GCImplementationRefs.InitRef));
+            XS.Call(LabelName.Get(GCImplementationRefs.InitRef));
+
+            WriteDebugVideo("Configuring Paging");
+            XS.Call("SystemVoidCosmosCorePagingInit");
```

Comment:
isn't there a way to get the label of a method dynamically? it may be useful to do that instead of this, just to avoid breaking it if there are other change on the future.

Reply:
Maybe changing it to:
```csharp
XS.Call(LabelName.Get(typeof(Paging).GetMethod(nameof(Paging.Init)));
```

Reply:
yeah there is a way to do that which I will do later